syntaxes/fracas.tmLanguage.json

```diff
@@ -58,7 +58,7 @@
       "patterns": [
         {
           "name": "keyword.define.fracas",
-          "match": "(?x) (?<=[\\(|\\{|\\[]) (define-enum|define-game-data|define-key|define-text|define-mask|define-type-optional|define-syntax|define-type|define-variant|define) (?=[\\)|\\}|\\]|\\s])\n"
```

Comment:
@thisisthedave I saw a lot of places in this file where the elements from `syntax.ts` seemed to show up - I think this file is probably autogenerated, but I wasn't sure how to trigger the generation of this `.json` file

Reply:
Unfortunately they're not generated from the same source, and consequently there is duplication. This json file is generated from another yml file though.

Reply:
https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide#textmate-grammars

Reply:
https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide

Reply:
The relevant bit for generating `tmLanguage.json` from `yaml`:
> VS Code can only load json grammars, so yaml based grammars must be converted to json. The [js-yaml package](https://www.npmjs.com/package/js-yaml) and command-line tool makes this easy.

```
# Install js-yaml as a development only dependency in your extension
$ npm install js-yaml --save-dev

# Use the command-line tool to convert the yaml grammar to json
$ npx js-yaml syntaxes/abc.tmLanguage.yaml > syntaxes/abc.tmLanguage.json
```

Reply:
FYI I already updated the yaml file and package.json